### PR TITLE
Enable `directly-loc-start-end` in `language-css`

### PR DESCRIPTION
**Pull request by fisker**
- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-css/utils.js [hunk 1]
```diff
@@ -461,7 +461,17 @@ function isAtWordPlaceholderNode(node) {
   );
 }
 
+function locStart(node) {
+  return node.source.startOffset;
```
Comment (thorn0) [inline src/language-css/utils.js:465]: Why not put these to `language-css/loc.js`?

Comment (fisker) [reply src/language-css/utils.js:465]: I was worried about Circular dependency in future.

Comment (fisker) [reply src/language-css/utils.js:465]: In `language-js`, `isBlockComment` function was in `comment.js`,when we  start use functions from `utils.js` in `comment.js`, I have to move it to `utils.js`.

Comment (thorn0) [reply src/language-css/utils.js]: On the other hand, files like `utils`, `helpers`, etc. are an antipattern. Usually it's better to move functions to other files with more meaningful names.

**Review by thorn0 — APPROVED** [review]

**Pull request merged**